src/ai/backend/common/events/events.py

```diff
@@ -1022,12 +1470,13 @@ def deserialize(cls, value: tuple) -> Self:
             VFolderID.from_str(value[0]),
         )
 
+    @classmethod
+    def event_name(cls) -> str:
+        return "vfolder_deletion_success"
 
-@attrs.define(auto_attribs=True, slots=True)
-class VFolderDeletionFailureEvent(AbstractEvent):
-    name = "vfolder_deletion_failure"
 
-    vfid: VFolderID
+@dataclass
+class VFolderDeletionFailureEvent(VFolderEvent):
```

Comment:
It's just question: Should all existing codes using `attrs.define` be refactored with dataclass? @HyeockJinKim